Projects/Chat/Sources/Views/InboxView.swift

```diff
@@ -41,24 +41,28 @@ public struct InboxView: View {
     }
 
     func rowViewContent(for conversation: Conversation) -> some View {
-        VStack(alignment: .leading, spacing: 8) {
-            HStack(alignment: .top) {
-                if conversation.type == .legacy {
-                    hText(L10n.chatConversationHistoryTitle, style: .body1)
-                        .foregroundColor(hTextColor.Opaque.primary)
-                } else {
-                    VStack(alignment: .leading, spacing: 0) {
-                        hText(conversation.title, style: .body1)
+        VStack(alignment: .leading, spacing: 0) {
+            VStack(alignment: .leading, spacing: 0) {
```

Comment:
do we really need duplicate of this VStack?

Reply:
maybe we do? or is there a "cleaner" way to do this?

Reply:
Yeah.... because of the paddings.
After inner VStack we want to add bottom at padding, and outer VStack says that that its a VStack
We could to 
```
 VStack(alignment: .leading, spacing: 0) {
            VStack(alignment: .leading, spacing: 0) {
```
to 
```
 VStack(alignment: .leading, spacing: 8) {
            VStack(alignment: .leading) {
```
And remove 
`.padding(.bottom, 8)`
Afterwards...

Reply:
maybe it's fine then 👍